test/ck_tile/gemm/test_gemm_pipeline.cpp

```diff
@@ -14,26 +14,28 @@ using Row       = ck_tile::tensor_layout::gemm::RowMajor;
 using Col       = ck_tile::tensor_layout::gemm::ColumnMajor;
 using Intrawave = ck_tile::integral_constant<ck_tile::GemmPipelineScheduler,
                                              ck_tile::GemmPipelineScheduler::Intrawave>;
-using Interwave = ck_tile::integral_constant<ck_tile::GemmPipelineScheduler,
-                                             ck_tile::GemmPipelineScheduler::Interwave>;
-using Mem       = ck_tile::integral_constant<GemmPipelineType, GemmPipelineType::Mem>;
-using Comp      = ck_tile::integral_constant<GemmPipelineType, GemmPipelineType::Comp>;
+// using Interwave = ck_tile::integral_constant<ck_tile::GemmPipelineScheduler,
+//                                              ck_tile::GemmPipelineScheduler::Interwave>;
+// using Mem       = ck_tile::integral_constant<GemmPipelineType, GemmPipelineType::Mem>;
+using Comp = ck_tile::integral_constant<GemmPipelineType, GemmPipelineType::Comp>;
+
+// TODO: Enable Memory pipeline, when it would be updated for vector loads on non-K major tensors.
 
 // clang-format off
 using KernelTypes = ::testing::Types<
     //         ALayout, BLayout, CLayout, ADataType, BDataType, AccDataType, CDataType, GemmPipelineScheduler, PipelineType
-    std::tuple<    Row,     Row,     Row,       F16,       F16,         F32,       F16,             Intrawave,         Mem>,
+    // std::tuple<    Row,     Row,     Row,       F16,       F16,         F32,       F16,             Intrawave,         Mem>,
```

Comment:
WHy commented?

Reply:
Cause Mem pipline is not yet updated to have vectorized reads on non-k-major tensors.